keras_cv/bounding_box/utils_test.py

```diff
@@ -38,3 +38,34 @@ def test_clip_to_image(self):
         )
         self.assertAllGreaterEqual(bboxes_out, 0)
         self.assertAllLessEqual(bboxes_out, 1)
+
+    def test_pad_with_sentinels(self):
+        bounding_boxes = tf.ragged.constant(
+            [[[1, 2, 3, 4, 5], [1, 2, 3, 4, 5]], [[1, 2, 3, 4, 5]]]
+        )
+        padded_bounding_boxes = bounding_box.pad_with_sentinels(bounding_boxes)
+        expected_output = tf.constant(
+            [
+                [[1, 2, 3, 4, 5], [1, 2, 3, 4, 5]],
+                [[1, 2, 3, 4, 5], [-1, -1, -1, -1, -1]],
+            ]
+        )
+        self.assertAllEqual(padded_bounding_boxes, expected_output)
+
+    def test_filter_sentinels(self):
```

Comment:
can we add a few more test cases with more complex inputs/outputs for filter sentinels?  some more differing shapes would be good.

Reply:
done